includes/plugin.php

```diff
@@ -700,6 +696,26 @@ private function register_autoloader() {
 		Autoloader::run();
 	}
 
+	/**
+	 * Plugin Magic Getter
+	 *
+	 * @since 3.1.0
+	 * @access public
+	 *
+	 * @param $property
+	 * @return mixed
+	 */
+	public function __get( $property ) {
+
+		if ( 'posts_css_manager' === $property ) {
+			self::$instance->modules_manager->get_modules( 'dev-tools' )->deprecation->deprecated_argument( 'posts_css_manager', '2.9.0', 'files_manager' );
+
+			return $this->files_manager;
+		}
+
+		return $this->property;
```

Comment:
Check this code with other private properties

Reply:
`Plugin` doesn't have any other private properties

Reply:
Good call. Checked it, had to modify. See new code.